### PR TITLE
Remove use of symlinks for tgui build

### PR DESCRIPTION
**Pull request by mordent-goonstation**
[CLEANLINESS][TGUI][TOOLING]

## About the PR 

Changes the build directory for tgui to `browserassets/tgui` rather than a random `public` folder within the `tgui` package, and associated changes around that.

## Why's this needed? 

Symlinks cause issues for Windows-based contributors.

### CODE REVIEW

File: tgui/.eslintignore [hunk 1]
```diff
@@ -1,6 +1 @@
 /**/node_modules
-/**/*.bundle.*
```
Comment (mordent-goonstation) [inline tgui/.eslintignore:2]: One advantage of moving the built files out of `tgui`, no need to exclude them from various things any more!

File: tgui/packages/tgui/webpack.config.js [hunk 2]
```diff
@@ -36,8 +36,8 @@ module.exports = (env = {}, argv) => {
     },
     output: {
       path: argv.useTmpFolder
-        ? path.resolve(__dirname, './public/.tmp')
-        : path.resolve(__dirname, './public'),
+        ? path.resolve(__dirname, '../../../browserassets/tgui/.tmp')
```
Comment (ZeWaka) [inline tgui/packages/tgui/webpack.config.js:39]: :kocmocpchela:

Comment (mordent-goonstation) [reply tgui/packages/tgui/webpack.config.js:39]: I am unclear if this is a request for code change.

File: tgui/.eslintignore (hunk 1 above)
Comment (ZeWaka) [reply tgui/.eslintignore:2]: 😸

**Pull request merged**